apps/condo/domains/billing/utils/serverSchema/checkAccountNumberWithOnlineInteractionUrl.js

```diff
@@ -17,18 +18,20 @@ const checkAccountNumberWithOnlineInteractionUrl = async (interactionUrl, tin, a
             timeoutBetweenRequests: 1000,
         })
         if (response.ok) {
-            const { status } = await response.json()
-            if (status === ONLINE_INTERACTION_CHECK_ACCOUNT_SUCCESS_STATUS) {
-                return true
-            }
-            logger.info({ msg: 'Billing account not found', payload: { url: url.origin, tin, accountNumber, status } })
+            return await response.json()
         }
     } catch (error) {
         logger.error({ err: error, payload: { url: url.origin, tin, accountNumber } })
     }
-    return false
+    return { status: ONLINE_INTERACTION_CHECK_ACCOUNT_ERROR_STATUS }
```

Comment:
strange output format, why status is optional field?

Reply:
status is not optional 
the function was divided into 2